### PR TITLE
Better sections for app containers

### PR DESCRIPTION
**Pull request by skjnldsv**
The Content doesn't really display properly, because this is of course not meant to be used within a page.
Shall we leave this like that?

Capture d’écran_2020-03-29_15-21-59

### CODE REVIEW

**Comment by skjnldsv** [comment]
Ping?

File: styleguide.config.js [hunk 1]
```diff
@@ -15,75 +15,94 @@ module.exports = {
...
+		},
 	}),
 	exampleMode: 'expand',
 	usageMode: 'expand',
 	components: 'src/components/*/*.vue',
 	getComponentPathLine(componentPath) {
-		const name = path.basename(componentPath, '.vue');
-		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}';`;
+		const name = path.basename(componentPath, '.vue')
+		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}';`
```
Comment (georgehrke) [inline styleguide.config.js:26]: We should remove the semicolon here was well :)

Comment (skjnldsv) [reply styleguide.config.js:26]:
```suggestion
		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}'`
```

**Comment by skjnldsv** [comment]
> It would be nice to show `Content` between `App containers` and `AppNavigation`, but this change is also good with me :)

I think it's sorted alphabetically, did not saw an order parametter in the vuestyleguide documentation :(

**Pull request merged**